spec/models/buy_opening_flow_spec.rb

```diff
@@ -141,8 +141,8 @@
     end
 
     it 'does not register litecoin buys' do
```

Comment:
This should be something along the lines of "does not register buys from another orderbook"